meet-data/nasa/todo/getpdfs.sh

```diff
@@ -3,16 +3,16 @@
 #One by one, it fetches the pdf, creates a new folder, moves the pdf to that folder, applies tabula to it
 #runs nasa-standardize to it, applies check-entries and stores the output.
 input="pdf_small"
-COUNTER=10
+COUNTER=1
 while IFS= read -r var
 do
-  mkdir /Users/blerner/openpowerlifting/meet-data/nasa/todo/"$COUNTER"
+  mkdir /Users/blerner/openpowerlifting/meet-data/nasa//140"$COUNTER"
```

Comment:
Three comments:

1. The `//` here is redundant, likely a typo.

2. It looks like this "140" is repeated in a bunch of places. Instead of repeating it, it would be better to use a variable.

3. "140" isn't correct, because `${COUNTER}` will be >= 10 for 2014. That will produce names like `14010` and `14011` instead of `1410` and `1411`. Really what you want to do is prepend `14` and left-zero-pad `${COUNTER}`.

Reply:
Ye this whole script is just for me really....

Reply:
Even if the script is just for you, it still seems like an odd logical error.